src/DataDepsGenerators.jl

```diff
@@ -12,26 +12,60 @@ abstract type DataRepo end
 
 const Opt{T} = Union{Missing, T}
 struct Metadata
+    shortname::Opt{String}
     fullname::Opt{String}
     website::Opt{String}
     description::Opt{String}
+    author::Opt{String}
```

Comment:
Lets keep all the authors.
So this can be an `Opt{Vector{String}}`
and then we can move `format_authors` out of each individual file